L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCConcentratorData.cc

```diff
@@ -0,0 +1,155 @@
+
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "DataFormats/L1THGCal/interface/HGCalConcentratorData.h"
+#include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
+#include "DataFormats/Common/interface/AssociationMap.h"
+#include "DataFormats/ForwardDetId/interface/HGCalTriggerDetId.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
+#include "L1Trigger/L1THGCalUtilities/interface/HGCalTriggerNtupleBase.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerTools.h"
+
+class HGCalTriggerNtupleHGCConcentratorData : public HGCalTriggerNtupleBase {
+public:
+  HGCalTriggerNtupleHGCConcentratorData(const edm::ParameterSet& conf);
+  ~HGCalTriggerNtupleHGCConcentratorData() override{};
+  void initialize(TTree&, const edm::ParameterSet&, edm::ConsumesCollector&&) final;
+  void fill(const edm::Event& e, const edm::EventSetup& es) final;
+
+private:
+  void clear() final;
+
+  HGCalTriggerTools triggerTools_;
+
+  edm::EDGetToken concentrator_data_token_;
+  edm::ESHandle<HGCalTriggerGeometryBase> geometry_;
+
+  static constexpr unsigned kPanelOffset_ = 0;
+  static constexpr unsigned kPanelMask_ = 0x7F;
+  static constexpr unsigned kSectorOffset_ = 7;
+  static constexpr unsigned kSectorMask_ = 0x7;
+
+  int econ_n_;
+  std::vector<uint32_t> econ_id_;
+  std::vector<int> econ_subdet_;
+  std::vector<int> econ_side_;
+  std::vector<int> econ_layer_;
+  std::vector<int> econ_panel_number_;
+  std::vector<int> econ_panel_sector_;
+  std::vector<int> econ_waferu_;
+  std::vector<int> econ_waferv_;
+  std::vector<int> econ_wafer_;
```

Comment:
Given the definition of the `HGCalConcentratorData` id, the `wafer` field  shouldn't exist